packages/rule-components/src/ReportDetails/RuleFeedback.js

```diff
@@ -2,19 +2,40 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ThumbsUpIcon from '@patternfly/react-icons/dist/js/icons/outlined-thumbs-up-icon';
 import ThumbsDownIcon from '@patternfly/react-icons/dist/js/icons/outlined-thumbs-down-icon';
+import { Button } from '@patternfly/react-core';
 
-const RuleFeedback = ({ ruleId }) => (
+// ruleId - is id of current rule
+// onFeedbackSent(ruleId, vote, text) is a callback which is called when feedback is changed
+// where ruleId is id of the rule, vote is either -1, 0 or 1 and text is a string
+const RuleFeedback = ({ ruleId, onFeedbackChanged }) => (
     <div>
-        <div>
-            <span>Is this rule helpful?</span>
+        <span>Is this rule helpful?</span>
+        <Button
+            className="ins-c-rule__rule-feedback-like-button"
+            variant="plain"
+            aria-label="Rule is helpful"
+            onClick={ () => {
+                onFeedbackChanged(ruleId, 1, '');
```

Comment:
Why do you have here 3 arguments, while use only 2?

Reply:
text feedback my be added at some point

Reply:
I think feedback should be done not in this component, this component is only two buttons, and some text, nothing more

Reply:
Why not? It's called RuleFeedback and I think it's the best place for text feedback as well. We can though remove third arg now because it's not used.